src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToRunILLink.cs

```diff
@@ -666,7 +666,60 @@ public void ILLink_verify_analysis_warnings_hello_world_app_trim_mode_copyused(s
         }
 
         [RequiresMSBuildVersionTheory("17.0.0.32901")]
-        [InlineData(ToolsetInfo.CurrentTargetFramework)]
+        [MemberData(nameof(Net6Plus), MemberType = typeof(PublishTestUtils))]
+        public void ILLink_verify_analysis_warnings_framework_assemblies(string targetFramework)
+        {
```

Comment:
I see that we have ILLink_verify_analysis_warnings_hello_world_app_trim_mode_copyused that does something very similar. My question is shouldn't that mode also run in Net6Plus (if we want to test warning differences between copyused/copy)